server/services/store/sqlstore/migrations/migrations_files/000016_subscriptions_table.up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS {{.prefix}}subscriptions (
+	block_type VARCHAR(10),
+	block_id VARCHAR(36),
+	workspace_id VARCHAR(36),
+	subscriber_type VARCHAR(10),
+	subscriber_id VARCHAR(36),
+	notified_at BIGINT,
+	create_at BIGINT,
+	delete_at BIGINT,
+	PRIMARY KEY (block_id, workspace_id, subscriber_id)
```

Comment:
Block ids are unique, so it shouldn't be necessary to add `workspace_id` to the PK

Reply:
Agreed.  That code was written before the unique block ids PR was merged. Thanks for catching that.

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/1851/commits/5ec12ade8b06a3799b7201d98b29da3dde8a10dc